pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp

```diff
@@ -90,6 +91,63 @@ class ConvertAtenOp : public OpConversionPattern<AtenOpT> {
 };
 } // namespace
 
+namespace {
+template <>
+LogicalResult ConvertAtenOp<OperatorOp>::matchAndRewrite(
+    OperatorOp op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  Location loc = op.getLoc();
+  auto name = op.name();
+  if (std::string("aten.einsum") == name) {
```

Comment:
Isn't  `aten.einsum` handled by official torch-mlir?

Reply:
Yes, `aten.einsum` haven't been added to `TorchDialect.td`